phpBB/includes/functions_display.php

```diff
@@ -486,10 +490,21 @@ function display_forums($root_data = '', $display_moderators = true, $return_mod
 
 				if ($subforum_row['display'] && $subforum_row['name'])
 				{
+					if ($subforum_row['type'] == FORUM_LINK && ($subforum_row['flags'] & FORUM_FLAG_LINK_TRACK) == false
+						&& empty($subforum_row['password']) && $auth->acl_get('f_read', $subforum_id))
+					{
+						$link = $subforum_row['link'];
```

Comment:
The idea was to directly show the target URL but we could of course just use the previous syntax and let view forum do the redirect